docs/00-getting-started.md

```diff
@@ -28,14 +28,17 @@ The minimum requirements to get started with Thunderbolt are:
     ```bash
     azb spark init
     ```
-    This will copy the default configuration files in config/ into a .thunderbolt/ directory in your current working directory.
+    This will create a *.thunderbolt* folder with preset configuration files in your current working directory.
 
 ### Setting up your accounts
+To finish setting up, you need to fill out your Azure Batch and Azure Storage secrets in *.thunderbolt/secrets.yaml*. We'd also recommend that you enter SSH key info in this file too.
+
+To get the required keys for Azure Batch and Azure Storage, please follow the below instructions:
```

Comment:
I would add screenshots here.

Reply:
of what? there are screenshots already.

Reply:
ohh.. missed them as they were hidden in the fold. nice!